main.rb

```diff
@@ -0,0 +1,54 @@
+require './app'
+
+def main
+  books = []
+  student = []
+  people = []
+  rental = []
+  option(books, student, people, rental)
+end
+
+def start_up
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Ceate a new book'
+  puts '5 - Ceate a new rental'
+  puts '6 - List all rental for given ID'
+  puts '7 - Exit'
+end
+
+def grab_choice(choice, books, people, rental)
+  case choice
+  when '1'
+    puts 'list of all books :'
+    display_list_of(books)
+  when '2'
+    puts 'List of all people:'
+    display_list_of_people(people)
+  when '3'
+    add_new_person(people)
+  when '4'
+    add_new_book(books)
```

Comment:
The same thing with choosing option 4 for creating a new book, I get an `ArguementError`.

![image](https://user-images.githubusercontent.com/38088818/181908295-a0159f2f-a9b5-4d05-9530-eb7552cc7d44.png)

- Kindly fix it.

* Since I cannot check `List all books` and `List all people` without creating a book and a person. After correcting the above-mentioned errors, be sure to check if books and people are listed properly.

* Aslo please verify if `creating a rental` and `listing rentals using id` works properly after books and people issues are resolved.